book/types_of_data.md

```diff
@@ -191,19 +223,54 @@ You can iterate over records by first transposing it into a table:
 ╰───┴──────┴───────╯
 ```
 
+Accessing records' data is similar to JavaScript:
```

Comment:
Comparing nushell to JavaScript may be helpful to people who know javascript but, for me, I'd rather see these comparisons left out. Others may feel different.

Reply:
I mention JavaScript mainly because records are derived from JSON (JavaScript Object Notation) and it's the nearest language with that syntactic device. But it's not especially necessary.